src/shared/utilities/localStorageUtil.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @description Abstracts the localStorage api
+ * This would ensure that all items saved on the localStorage are uniform
+ */
+class localStorageUtil {
+  /**
+   * @description Retrieves an item from the localStorage
+   * @param {String} name The name of the object to be retrieved
+   * @returns {Object}
+   */
+  static getItem(name) {
+    const result = localStorage.getItem(name);
+    return result === 'undefined' ? null : JSON.parse(result);
+  }
```

Comment:
@chukwuemekachm when dealing with state objects, it is best practice to return undefined when the object does not exist, this way, if it is a default object, the reducers will instantiate it on load, otherwise, the object is absent in the state and does not lead to `cannot read property of null` errors.